python-avd/pyavd/_eos_designs/structured_config/overlay/router_bgp.py

```diff
@@ -216,9 +216,16 @@ def _address_family_evpn(self: AvdStructuredConfigOverlay) -> dict | None:
         if self.shared_utils.overlay_evpn_vxlan is True:
             if self.shared_utils.is_wan_router:
                 overlay_peer_group_name = self.shared_utils.bgp_peer_groups["wan_overlay_peers"]["name"]
+                peer_groups.append(
+                    {
+                        "name": overlay_peer_group_name,
```

Comment:
```suggestion
                peer_groups.append(
                    {
                        "name": self.shared_utils.bgp_peer_groups["wan_overlay_peers"]["name"],
```

Similar below (github won't let me suggest there)

Reply:
we cant do it like this as this is required later to check if overlay_peer_group_name has been set. Let me see if I can refactor this better